src/redux/features/observations/observationMonitoringPlotSelectors.ts

```diff
@@ -22,5 +22,5 @@ export const selectObservationMonitoringPlot = createSelector(
           plantingSubzoneName: subzone.plantingSubzoneName,
         }))
       )
-      .find((monitoringPlot: PlotObservations) => monitoringPlot.monitoringPlotId === params.monitoringPlotId)
+      .find((monitoringPlot: PlotObservations) => monitoringPlot.monitoringPlotId === params.monitoringPlotId) ?? null
```

Comment:
Is this necessary? Won't `find` return undefined anyways if its predicate fails to find a match?